### ISSUE TITLE
How to generate attn_weights_7b/headmask_24.npy

### ISSUE THREAD

**Issue by tanghao2118** [ZhangAIPI/YOPO_MLLM_Pruning#15]
`
self.mask = torch.tensor(np.load("attn_weights_7b/headmask_24.npy")).cuda()
`

**Comment by zhaowt61** [comment]
Please take a look at this ticket https://github.com/ZhangAIPI/YOPO_MLLM_Pruning/issues/1
